server/routes/support/handlers/probationStaff.ts

```diff
@@ -31,7 +30,7 @@ export default class ProbationTeamRoutes {
         : await this.comCaseloadService.getStaffVaryCaseload({ ...user, deliusStaffIdentifier: staff.staffIdentifier })
 
     return res.render('pages/support/probationStaff', {
-      caseload: createCaseloadViewModel(caseload),
+      caseload,
```

Comment:
**question:** Do we not need to add the additional fields added in `handlers/caseload.ts` to the caseload object here to match the COM caseload view?

Reply:
no, we don't use any of those fields on this view. This view is quite limited as it displays both vary and create cases and only uses fields common to both of them